Automattic/vip-go-mu-plugins
# Skip confirm works on --setup

## Description


## Changelog Description

### Plugin Updated: Enterprise Search

Running `wp index --setup --skip-confirm` no longer prompts for the second prompt.

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1) Run: `vip dev-env exec -- wp vip-search index --skip-confirm  --setup`
2) No prompt confirmation is required
